ImperatorToCk3/Source/Ck3/Ck3World.h

```diff
@@ -0,0 +1,29 @@
+#ifndef CK3_WORLD
+#define CK3_WORLD
+
+
+
+#include "../Imperator/ImperatorWorld.h"
+
+class Configuration;
+
+namespace Ck3World
+{
+
+class World
+{
+	public:
+		World(const ImperatorWorld::World& impWorld, const Configuration& theConfiguration);
+		World(const ImperatorWorld::World& impWorld): modName(impWorld.getSaveName()) {};
+
+		std::string getModName() const { return modName; }
```

Comment:
modname is vague. It can mean incoming mods or outgoing mod name. Use something more specific please.